### ISSUE TITLE
[a11y]: test to be deleted

### ISSUE THREAD

**Issue by tay1orjones** [carbon-design-system/carbon#20293]
### Package

@carbon/icons-vue

### Browser

Firefox

### Operating System

MacOS

### Package version

v11

### React version

v19

### Automated testing tool and ruleset

n/a

### Assistive technology

JAWS

### Description

when listing `@carbon/themes` the label should be applied

### WCAG 2.1 Violation

n/a

### Reproduction/example

n/a

### Steps to reproduce

QRadar XDR/EDR (ReaQta)

### Suggested Severity

Severity 1 = Must be fixed ASAP. The response must be swift. Someone from the team must drop all current work and be immediately reassigned to address the issue.

### Code of Conduct

- [x] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon/blob/f555616971a03fd454c0f4daea184adf41fff05b/.github/CODE_OF_CONDUCT.md)
- [x] I checked the [current issues](https://github.com/carbon-design-system/carbon/issues) for duplicate problems
